### PR TITLE
Switch csv-parse for papaparse

### PR DESCRIPTION
**Pull request by zakpatterson**
csv-parse is causing too many issues with being based as a node.js project and polyfilling for browser, papaparse seems to be simpler.

### CODE REVIEW

File: src/components/income/TransactionImporter.tsx [hunk 1]
```diff
@@ -126,11 +127,15 @@ export const TransactionImporter = (): ReactElement => {
 
   const onHandle = async (contents: string): Promise<void> => {
     setRawContents(contents)
-    const res = await preflightCsv(contents).catch((parseError) => {
-      console.error("Couldn't parse CSV", parseError)
-      return []
-    })
-    setPreflightTransactions(res)
+    run(preflightCsv(contents)).fold(
```
Comment (zakpatterson) [inline src/components/income/TransactionImporter.tsx:130]: both of these should be void

Comment (zakpatterson) [reply src/components/income/TransactionImporter.tsx:130]: Hm I can't "request changes" on my own PR.

**Comment by zakpatterson** [comment]
Errors are not showing if there's an unparsable row...fixing.

**Review by thegrims — APPROVED** [review]
LGTM

**Pull request merged**